packages/shared/src/libs/helpers/get-sync-time/get-sync-time.helper.ts

```diff
@@ -0,0 +1,31 @@
+import { differenceInHours, formatDistance } from "date-fns";
+
+type SyncTimeResult = {
+	hoursDifference?: number;
+	label: string;
+};
+
+const getSyncTime = (date: Date, baseDate: Date): SyncTimeResult => {
+	const HOURS_IN_A_DAY = 24;
+	const ONE_HOUR = 1;
+	const ZERO_HOURS = 0;
```

Comment:
I think you can take the constants beyond the function
```suggestion
const HOURS_IN_A_DAY = 24;
const ONE_HOUR = 1;
const ZERO_HOURS = 0;

const getSyncTime = (date: Date, baseDate: Date): SyncTimeResult => {
```